src/components/block-css/index.js

```diff
@@ -338,11 +338,13 @@ const BlockCss = props => {
 			blockUniqueClassName = blockUniqueClassName + `-${ instanceId }`
 		}
 
+		if ( ! Array.isArray( selector ) ) {
```

Comment:
Since the fix seems to be about fixing the error when the `replace` function is not available, maybe the `if` statement should be only when `selector` is a `string` instead of checking if it's **not** an array.